moto/panorama/models.py

```diff
@@ -0,0 +1,304 @@
+import base64
+import json
+from datetime import datetime, timedelta
+from dateutil.tz import tzutc
+from typing import Dict, Any, Optional, Union, List
+from typing_extensions import Literal
```

Comment:
Moto doesn't have the `typing_extensions` dependency (at runtime), and I don't really want to introduce it either.

Can we replace these all just with `attr: str`? If necessary, just leave a comment with the available/allowed values.

Reply:
Done: https://github.com/getmoto/moto/pull/6948/commits/ff562d22b7a6f5bdfba5bc7b5e1dc074dfd50934
Sorry I did this In the same commit